app/views/posts/index.html.erb

```diff
@@ -8,7 +8,7 @@
 <% end %>
 
 <ul class="[&>*:nth-child(odd)]:bg-green-500 [&>*:nth-child(odd)]:text-violet-100 [&>*:nth-child(odd):hover]:border-green-700 [&>*:nth-child(even):hover]:border-blue-400 [&>*:nth-child(even)]:bg-blue-200">
-  <% @posts.each do |post| %>
+  <% @posts.includes(:comments).each do |post| %>
     <li class="hover:cursor-pointer rounded-xl m-8 p-8 border-4 border-transparent">
```

Comment:
- [ ] Instead of fixing the n+ problem here, Please make sure to fix it in the `posts` controllers.  It's not a professional and good practice to fix the n+ problem in the views.
  - You can consider using the below code.

`@posts = @user[0].posts.includes(:comments)` instead of `@posts = @users[0].posts`